pipeline/00-ingest.R

```diff
@@ -270,12 +272,12 @@ training_data_klg <- training_data_ms %>%
 # likely non-arms-length sales. ONLY APPLIES to multi-sale properties
 training_data_fil <- training_data_klg %>%
   mutate(
-    sv_outlier_type = case_when(
+    sv_outlier_reason1 = case_when(
       meta_sale_price < 50000 & meta_sale_num_parcels == 2 ~
```

Comment:
**question:** Have we updated the condo sales val such that it can handle limited multi-sales yet? If not, how do we feel about this kludge and its bounds? @wagnerlmichael @jeancochrane